.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thank you for helping to improve Upbound!
+
+Please read through https://git.io/fj2m9 if this is your first time opening an
+Upbound pull request. Find us in https://slack.crossplane.io/messages/upbound if
+you need any help contributing.
+-->
+
+### Description of your changes
+
+<!--
+Briefly describe what this pull request does. Be sure to direct your reviewers'
+attention to anything that needs special consideration.
+
+We love pull requests that resolve an open issue. If yours does, use the below
+line to indicate which issue your PR fixes, for example "Fixes #500":
+-->
+
+Fixes #
+
+I have:
+
+- [ ] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
+- [ ] Run `make reviewable test` to ensure this PR is ready for review.
+- [ ] Added `backport-release-x.y` labels to auto-backport this PR.
```

Comment:
We could use https://mergify.io/ to automate the creation and merging of backport PRs for any PR with this label, which would reduce some of the toil involved in releasing bug fixes. @jbw976 recommended this tool based on its use in the @rook project. See https://github.com/rook/rook/pull/7972 as an example.

Note that Mergify is $8/mo/user for orgs with private repos like @upbound. There appear to be similar Actions available for free, for example https://github.com/tibdex/backport.

Reply:
After much trial and error I landed on an action that works well.